packages/react/src/emotion-element.js

```diff
@@ -37,21 +33,17 @@ export const createEmotionProps = (type: React.ElementType, props: Object) => {
 
   newProps[typePropName] = type
 
-  if (process.env.NODE_ENV !== 'production') {
-    const error = new Error()
-    if (error.stack) {
-      // chrome
-      let match = error.stack.match(
-        /at (?:Object\.|Module\.|)(?:jsx|createEmotionProps).*\n\s+at (?:Object\.|)([A-Z][A-Za-z0-9$]+) /
-      )
-      if (!match) {
-        // safari and firefox
-        match = error.stack.match(/.*\n([A-Z][A-Za-z0-9$]+)@/)
-      }
-      if (match) {
-        newProps[labelPropName] = sanitizeIdentifier(match[1])
-      }
-    }
+  // For performance, only call getLabelFromStackTrace in development and when
+  // the label hasn't already been computed
+  if (
+    process.env.NODE_ENV !== 'production' &&
+    !!props.css &&
```

Comment:
We already know that this is truthy here - the first `if` in this function already assumes that and it works
```suggestion
```

Reply:
pretty sure this is needed, the check at the start of the function checks if it's a string first and then does things. The places where this function is called only check that props has an own prop named `css`

Reply:
ah, ok - makes sense, we indeed allow this `css={null}`